autoload/wiki/journal.vim

```diff
@@ -53,6 +53,43 @@ function! wiki#journal#go_to_month() abort " {{{1
   call wiki#url#parse('journal:' . l:date[:3] . '_m' . l:date[5:6]).open()
 endfunction
 
+" }}}1
+function! wiki#journal#make_index() " {{{1
```

Comment:
Could we add an argument to support different link styles? Something like `make_index(use_md_links)`?